front/lib/api/assistant/generation.ts

```diff
@@ -468,6 +468,9 @@ export async function constructPromptMultiActions(
     }
   }
 
+  additionalInstructions +=
+    "When generating latex formulas, solely rely on the $$ escape sequence, single $ latex sequence are not supported.\n";
```

Comment:
I would put one `\n` before as well out of consistency with the above but no strong feeling

Reply:
Also my bad but I believe it should be

"single $ latex sequenceS" but to be double checked

Reply:
Some have only an ending `\n`, some have both indeed.
 I added both to mark it's unrelated to the previous instruction!
I also fixed the typo, thanks 🙏🏻